Aztec/Classes/Public/Extensions/NSAttributedString+Attachments.swift

```diff
@@ -22,4 +22,26 @@ extension NSAttributedString
             }
         }
     }
+
+    /// Determine the character ranges for an attachment
+    public func ranges(forAttachment attachment: NSTextAttachment) -> [NSRange]
+    {
+        // find character range for this attachment
```

Comment:
IMHO we can remove this comment - the method documentation should state what we're doing in here anyway.